src/psyclone/domain/lfric/lfric_types.py

```diff
@@ -135,16 +137,19 @@ def init():
     # ------------------------------------------------------------------------
     @staticmethod
     def _create_precision_from_const_module():
-        '''This function implements all precisions defined in
-        ``LFRicConstants.PRECISION_MAP``. It adds "constants_mod" as
+        '''This function implements all precisions defined in the default
```

Comment:
The fact that this happens to be the default is probably (?) not relevant. I'd just go for "...defined for the `dynamo0.3` (LFRic) domain."